### DISCUSSION TITLE
Add Expanded/Collapsed option to Field Group

### DISCUSSION THREAD

**Discussion by Bodom78** [directus/directus#6829] [Feature Requests]
Just started using the new Field Group Interface which is great but the fact that all divider groups are collapsed by default is a little restrictive.

Would be good to have the option to set their expanded/collapsed state. 

Perhaps even integrate into the Presets & Bookmarks?

**Comment by benhaynes** [comment]
Absolutely. I think we have a few comments about doing this, and we certainly will. Just have to take care of a few other priorities first, unless somebody wants to open a pull request.

**Reply by shadowcpy** [reply]
Will fork and try.
@Bodom78 Regarding the Presets & Bookmarks. I would add the option of default state in the group field flow / definitions. The Presets and bookmarks feature is for Collection view, not detail view, afaik.

**Comment by shadowcpy** [comment]
#7059 PR created
